src/ripple/app/ledger/impl/MetaView.cpp

```diff
@@ -482,23 +533,212 @@ Json::Value MetaView::getJson (int) const
 
     ret[jss::nodes] = nodes;
 
-    ret[jss::metaData] = mSet.getJson (0);
+    // VFALCO The meta only exists during apply() now
+    //ret[jss::metaData] = meta.getJson (0);
 
     return ret;
 }
 
 //------------------------------------------------------------------------------
 
+void
+MetaView::apply (BasicView& to,
+    STTx const& tx, TER ter, beast::Journal j)
+{
+    auto const sTx =
+        std::make_shared<Serializer>();
+    tx.add(*sTx);
+
+    std::shared_ptr<Serializer> sMeta;
+
+    if (closed())
+    {
+        TxMeta meta;
+        // VFALCO Shouldn't TxMeta ctor do this?
+        meta.init (tx.getTransactionID(), seq());
+        if (deliverAmount_)
+            meta.setDeliveredAmount(
+                *deliverAmount_);
+
+        Mods newMod;
+        for (auto& it : items_)
+        {
+            auto type = &sfGeneric;
+            switch (it.second.first)
+            {
+            case taaMODIFY:
+            #ifdef META_DEBUG
+                JLOG(j.trace) << "modify " << it.first;
+            #endif
```

Comment:
Consider defining a macro for this to remove all the `ifdef`'s in this code: `META_DEBUG_JLOG(j, "modify", it.first)` that expands to the calls here (with a shorter name, of course).